### ISSUE TITLE
Can't reach reported accuracy on MiniImagenet dataset of baseline++ with ResNet18

### ISSUE THREAD

**Issue by bl0** [wyharveychen/CloserLookFewShot#40]
Hello, Thanks for your great work. 
I have run your code with the following command for several times, but the final accuracy is all about 73.99, which is slower than 75.68 reported in the paper. Am I missing something? Or I have to tune some hyperparameters?

```bash
for file in train save_features test; do
    python ./${file}.py --dataset miniImagenet --model ResNet18 --method baseline++ --train_aug --n_shot 5
done
```

Thanks very much.

**Comment by liuzechun** [comment]
Dear Author,
Thanks a lot for your work! 
I also got the same issue with ResNet10 baseline++ on miniImageNet.
I run:
`python ./train.py --dataset miniImagenet --model ResNet10 --method baseline++ --train_aug`
But got 74.10% +- 0.62%, instead of  75.90±0.61. Is there anything I can adjust?
Thanks!

**Comment by XuChunqiao** [comment]
Hello, Thanks for your great work.
I have the same problem .
I run:
python ./train.py --dataset miniImagenet --model ResNet18 --method baseline++ --train_aug
python ./save_features.py --dataset miniImagenet --model ResNet18 --method baseline++ --train_aug
python ./test.py --dataset miniImagenet --model ResNet18 --method baseline++ --train_aug
Final accuracy is 74.38% +- 0.64% instead of 75.68% +- 0.64%.Did I miss something?
Thank you very much.

**Comment by janetwise** [comment]
Maybe try to increase batch_size in the training. I used 128 as suggested in another issue and got 5%+ higher accuracy. You can try 32, 64... (default 16) based on your gpu mem.
